mavenによるbuildにおけるLombok対応

### １．pom.xml
以下のdependencyが、２つ連続で定義されていました。mavenのbuildでワーニングが出てまして。
片方を消しておいたほうがよいですね。
```
    <dependency>
      <groupId>org.springframework.boot</groupId>
      <artifactId>spring-boot-configuration-processor</artifactId>
      <optional>true</optional>
    </dependency>
```

### 2.mavenでbuildエラー
以下のようなbuildエラーが大量に発生して、buildが失敗しています。
```
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /C:/Users/mogi/git/Training/sample/jsug-shop/src/test/java/jsug/domain/repository/account/AccountRepositoryTest.java:[32,34] シンボルを見つけられません
  シンボル:   メソッド builder()
  場所: クラス jsug.domain.model.Account
```
jsug-shopではLombokというライブラリを使用し、getter/setter等のメソッドを内部的に自動生成している。コンパイル時に自動生成するが、mavenによるbuildでLombokが認識されていないことから、各メソッドが存在しないと認識されてしまう。
解決方法としては、以下が考えられる。
1. spring-boot-maven-pluginをpom.xmlに組み込む。
2. mavenプロジェクトをupdate
3. maven　clean
4. maven install

茂木のlocalでfixを試みます。うまくいったら、pull requestを出します。

--- a/sample/jsug-shop/pom.xml
+++ b/sample/jsug-shop/pom.xml
@@ -68,11 +68,6 @@
       <artifactId>spring-boot-configuration-processor</artifactId>
       <optional>true</optional>
     </dependency>
-    <dependency>
-      <groupId>org.springframework.boot</groupId>
-      <artifactId>spring-boot-configuration-processor</artifactId>
-      <optional>true</optional>
-    </dependency>
 
     <!-- war生成時に組み込みTomcatを除外する設定。 -->
     <!-- warのファイルサイズを削減するために、必須と言って良い。 -->

